apps/functions-subscription/api/internal.yaml

```diff
@@ -329,3 +369,36 @@ components:
     SubscriptionEvent:
       type: object
       $ref: '#/components/schemas/Subscription'
+    CreateTrial:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          $ref: '#/components/schemas/TrialName'
+        description:
+          $ref: '#/components/schemas/TrialDescription'
+    Trial:
+      type: object
+      required:
+        - trialId
+        - name
+        - state
+      properties:
+        trialId:
+          $ref: '#/components/schemas/TrialId'
+        name:
+          $ref: '#/components/schemas/TrialName'
+        description:
+          $ref: '#/components/schemas/TrialDescription'
+        state:
+          type: string
+          description: |-
+            - INACTIVE: The trial exists, but it has not started yet.
+            - ACTIVE: The trial is active.
+            - ENDED: The trial has ended.
+          enum:
+            - INACTIVE
+            - ACTIVE
+            - ENDED
+
```

Comment:
**suggestion (non-blocking)**: If it is not required at the moment, consider removing it.

Reply:
I'd keep the `state` property, because it has been requested in a comment on the requirements.
Even if we do not have the requirement to update the trial (e.g. set it as `ended`)

Reply:
I changed my mind and I decided to remove the `state` property. I think we should add this when there is the requirement